src/main/java/net/dries007/tfc/util/ICollapsableBlock.java

```diff
@@ -5,8 +5,65 @@
 
 package net.dries007.tfc.util;
 
+import net.minecraft.block.BlockFalling;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import net.dries007.tfc.objects.entity.EntityFallingBlockTFC;
+
 // For raw stone, because it collapses
-public interface ICollapsableBlock
+public interface ICollapsableBlock extends IFallingBlock
 {
-    // todo: this
+    default void checkFalling(World worldIn, BlockPos pos, IBlockState state)
```

Comment:
This needs discussion + I need to test to see exactly how you've implemented this.